ui/hooks/featureflags.ts

```diff
@@ -5,5 +5,12 @@ export type FeatureFlags = { [key: string]: string };
 
 export function useFeatureFlags() {
   const { featureFlags } = useContext(CoreClientContext);
-  return { data: { flags: featureFlags || {} } };
+
+  const isFlagEnabled = (flag: string) => {
+    if (flag === "WEAVE_GITOPS_FEATURE_OIDC_BUTTON_LABEL") {
+      return featureFlags?.["WEAVE_GITOPS_FEATURE_OIDC_BUTTON_LABEL"] !== "";
+    } else return featureFlags?.[flag] === "true";
+  };
+
+  return { isFlagEnabled };
```

Comment:
WDYT about still exposing `flags` so we don't have to special case things in the `isFlagEnabled` helper?

- Most flag logic can use `isFlagEnabled` as its usually a string-bool
- Some flags will have other values like labels in them and they can still be accessed via `flags`?

```suggestion
  return { isFlagEnabled, flags };
```